<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecificEntityConfigurationDialog</class>
 <widget class="QDialog" name="SpecificEntityConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specific Configuration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Specific configuration</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Percentage of Silhouette dots
 that suffer dot dispersion:
(0% by default)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Percentage of internal dots
(dots that overlap the 3D model)
 that are generated:
 (Overrides general 
 configuration if different of -1)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="percentageSilhouetteDispersion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="percentageInternalGeneration">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="edgeDetectionMethod">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge detection method:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpecificEntityConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpecificEntityConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
